docs/alerts.md

```diff
@@ -23,7 +23,85 @@ At least one primary shard and its replicas are not allocated to a node.
 
 ### Troubleshooting
 
-Cluster nodes could be failed or the Elasticsearch process crashes due to heavy load.
+1. Confirm if the elasticsearch cluster’s health is RED:
+   ```
+   oc exec -n openshift-logging -c elasticsearch <elasticsearch_pod_name> -- es_cluster_health
+   ```
+   Check for the “status” field in the output of the above command.
+
+2. Check the nodes that have joined the cluster:
+   ```
+   oc exec -n openshift-logging -c elasticsearch <elasticsearch_pod_name> -- es_util --query=_cat/nodes?v
+   ```
+   Also, check the elasticsearch pods:
+   ```
+   oc -n openshift-logging get pods -l component=elasticsearch
+   ```
+
+3. If not all nodes have joined the cluster:
+    - Check for the elected master:
+      ```
+      oc exec -n openshift-logging -c elasticsearch <elasticsearch_pod_name> -- es_util --query=_cat/master?v
+      ```
+    - Check master pod logs:
+      ```
+      oc logs <elasticsearch_master_pod_name> -c elasticsearch -n openshift-logging
+      ```
+    - Check logs of nodes that haven’t joined the cluster:
+      ```
+      oc logs <elasticsearch_node_name> -c elasticsearch -n openshift-logging
+      ```
+
```

Comment:
We probably should have a follow up step here... but this starts to get really tricky... @lukas-vlcek can you think of some steps we can take here? Do we just want to try to restart the nodes that haven't joined? but if there's a cert issue we need to figure out which one has the correct certs... also the operator should be doing something there already...